### DISCUSSION TITLE
1.5.0rc3

### DISCUSSION THREAD

**Discussion by villebro** [apache/superset#19739] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.5.0. New cherries since 1.5.0rc2:
- #19651 fix(dnd&column): make to fix the blank state issue when only one column select (@prosdev0107)
- #19716 fix: drop the first level of MultiIndex (@zhaoyongjie)
- #19688 fix(dashboard): Fix BigNumber causing dashboard to crash when overflowing (@kgabryje)
- #19664 fix(sql-lab): do not replace undefined schema with empty object (@villebro)
- #19659 fix: time comparision (@zhaoyongjie)
- #19636 chore: skip SIP-68 shadow writing for LTS (@ktmud)

The release notes (unchanged since rc2):
https://github.com/apache/superset/tree/master/RELEASING/release-notes-1-5

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.0rc3/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.0rc3

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.0rc3/CHANGELOG.md

The Updating instructions for the release (unchanged since rc1):
https://github.com/apache/superset/blob/1.5.0rc3/UPDATING.md

public keys are available at:
https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.5.0rc3'>1.5.0rc3</a>.</em>

**Comment by zhaoyongjie** [comment]
+1 binding

On Sat, Apr 16, 2022 at 15:26 Ville Brofeldt ***@***.***>
wrote:

> Hello Superset Community,
>
> This is a call for the vote to release Apache Superset version 1.5.0. New
> cherries since 1.5.0rc2:
>
>    - #19651 <https://github.com/apache/superset/pull/19651>
>    fix(dnd&column): make to fix the blank state issue when only one column
>    select ***@***.*** <https://github.com/prosdev0107>)
>    - #19716 <https://github.com/apache/superset/pull/19716> fix: drop the
>    first level of MultiIndex ***@***.***
>    <https://github.com/zhaoyongjie>)
>    - #19688 <https://github.com/apache/superset/pull/19688>
>    fix(dashboard): Fix BigNumber causing dashboard to crash when overflowing (
>    @kgabryje <https://github.com/kgabryje>)
>    - #19664 <https://github.com/apache/superset/pull/19664> fix(sql-lab):
>    do not replace undefined schema with empty object ***@***.***
>    <https://github.com/villebro>)
>    - #19659 <https://github.com/apache/superset/pull/19659> fix: time
>    comparision ***@***.*** <https://github.com/zhaoyongjie>)
>    - #19636 <https://github.com/apache/superset/pull/19636> chore: skip
>    SIP-68 shadow writing for LTS ***@***.*** <https://github.com/ktmud>)
>
> The release notes (unchanged since rc2):
> https://github.com/apache/superset/tree/master/RELEASING/release-notes-1-5
>
> The release candidate:
> https://dist.apache.org/repos/dist/dev/superset/1.5.0rc3/
>
> Git tag for the release:
> https://github.com/apache/superset/tree/1.5.0rc3
>
> The Change Log for the release:
> https://github.com/apache/superset/blob/1.5.0rc3/CHANGELOG.md
>
> The Updating instructions for the release (unchanged since rc1):
> https://github.com/apache/superset/blob/1.5.0rc3/UPDATING.md
>
> public keys are available at:
> https://www.apache.org/dist/superset/KEYS
>
> The vote will be open for at least 72 hours or until the necessary number
> of votes are reached.
>
> Please vote accordingly:
>
> [ ] +1 approve
> [ ] +0 no opinion
> [ ] -1 disapprove with the reason
>
> Thanks,
> The Apache Superset Team
> ------------------------------
> *This discussion was created from the release 1.5.0rc3
> <https://github.com/apache/superset/releases/tag/1.5.0rc3>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/superset/discussions/19739>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAPMKUS6424KDJ7VXNJMUWDVFJTT3ANCNFSM5TSABZLA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 

Best regards,

Yongjie

**Comment by kamalkeshavani-aiinside** [comment]
@villebro Any chance [this](https://github.com/apache/superset/issues/18959) can be fixed or we can remove this warning `filter_box will be deprecated in a future version of Superset. Please replace filter_box by dashboard filter components.`

**Reply by villebro** [reply]
> @villebro Any chance [this](https://github.com/apache/superset/issues/18959) can be fixed or we can remove this warning `filter_box will be deprecated in a future version of Superset. Please replace filter_box by dashboard filter components.`

@kamalkeshavani-aiinside unfortunately I haven't been able to work on this issue. @amitmiran137 if you have some spare cycles, would you be interested in checking this out? Shouldn't be a super difficult fix.
